app/components/views/TicketsPage/MyTicketsTab/MyTicketsTab.jsx

```diff
@@ -112,6 +131,59 @@ const MyTickets = ({ toggleIsLegacy }) => {
     setSortOrderKey(type.value);
   };
 
+  const arraysEqual = (a, b) => {
```

Comment:
all of this should move to the custom hook above `useTicketsList` and shouldn't be part of the component.